### DISCUSSION TITLE
How do I link GitHub account to my EPIC GAMES account?

### DISCUSSION THREAD

**Discussion by chevdave** [community/community#145825] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I link GitHub account to my EPIC GAMES account?

**Comment by CyberSphinxxx** [comment]
Hi there @chevdave!

Linking your GitHub account to your Epic Games account is a straightforward process. Here’s how you can do it:  

---

### 1. **Go to Epic Games Account Settings**  
- Log in to your Epic Games account at [Epic Games Account Settings](https://www.epicgames.com/account).  

### 2. **Find the Connections Tab**  
- On the left-hand side, click on **"Apps & Accounts"** tab.  

### 3. **Connect Your GitHub Account**  
- Look for the **GitHub** option in the list of accounts. 
- Read the EULA and **"Agree"**
- Click **"Connect"**, and a pop-up will appear asking you to log in to your GitHub account (if you’re not already logged in).  

### 4. **Authorize the Connection**  
- After logging into GitHub, you’ll be prompted to authorize Epic Games to access your GitHub account.  
- Click **"Authorize"** to complete the process.  

### 5. **Confirmation**  
- Once connected, you should see a confirmation in your Epic Games account settings.  

---
### Note: 
- Check your browser for pop-up blockers. They might prevent the connection screen from appearing.  

## Screenshot
![Opera Snapshot_2024-11-29_112142_www epicgames com](https://github.com/user-attachments/assets/2d09db13-24d6-4bc0-aebd-994ba240008d)

I hope this helps!

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @chevdave!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
